extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventDataFetchFieldsMixin.json

```diff
@@ -0,0 +1,96 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEventDataFetchFields",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Data Fetch Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEvent",
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Journey Orchestration Data Fetch Field for a Step Event",
+  "definitions": {
+    "journeyStepEventDataFetchFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Step Event Data Fetch Field",
+          "type": "object",
+          "properties": {
+            "journey:fetchTotalTime": {
+              "title": "fetchTotalTime",
+              "description": "This Field describes the total amount of time spent in data fetch in millis during the step processing",
+              "type": "integer",
+              "minimum": -9007199254740991,
+              "maximum": 9007199254740991
+            },
+            "journey:fetchTypeInError": {
+              "title": "fetchTypeInError",
+              "description": "This Field defines if the fetch in error is on the platform or on a custom data source",
```

Comment:
If there is a limited set of values, make it an enum

Reply:
there can be any number of custom data sources and their respective errors  which are unknown to us right now so used string there